### PR TITLE
Added a top rating view

### PR DESCRIPTION
**Pull request by Cheshiriks**
- added a top rating view

Rating

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/toprating/OrganizationRatingTab.kt [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+import react.dom.html.ReactHTML.th
+import react.dom.html.ReactHTML.tr
+import react.router.dom.Link
+import web.cssom.*
+
+import kotlinx.browser.window
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+val renderOrganizationRatingTab: FC<Props> = FC { _ ->
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/toprating/OrganizationRatingTab.kt:30]: To my mind, `render` here is odd. Should be renamed to `organizationRatingTab`

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/toprating/TopRatingView.kt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            title(" Top Rating", faTrophy)
+        }
+    }
+
+    tab(selectedMenu.name, TopRatingTab.values().map { it.name }, "nav nav-tabs mt-3") { value ->
+        setSelectedMenu(TopRatingTab.valueOf(value))
+    }
+
+    when (selectedMenu) {
+        TopRatingTab.USERS -> renderUserRatingTable()
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/toprating/TopRatingView.kt:40]: Here also there is no need to add `render` as it is not a function that renders, but a `val` that is a tab

**Review by sanyavertolet — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/toprating/OrganizationRatingTab.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/toprating/OrganizationRatingTab.kt]: Ok. Renamed it :)

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/toprating/TopRatingView.kt (hunk 2 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/toprating/TopRatingView.kt]: Fixed

**Pull request merged**